libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html

```diff
@@ -1,16 +1,11 @@
-<sky-input-box
-  *ngIf="isAlive && useAsyncSearch"
-  [disabled]="skyComponentProperties?.disabled"
-  [formGroup]="editorForm"
->
+<sky-input-box *ngIf="isAlive && useAsyncSearch" [formGroup]="editorForm">
```

Comment:
I'm assuming we don't need to set this anymore because of the easy mode work?

Reply:
I believe this was an error from cherry-picking the old PR. Removed this change.